lib/active_interactor/context/attributes.rb

```diff
@@ -132,7 +132,7 @@ def merge!(context)
         copy_flags!(context)
 
         merged_context_attributes(context).each_pair do |key, value|
-          public_send("#{key}=", value) unless value.nil?
+          self[key] = value unless value.nil?
```

Comment:
I'm curious why this works... If we look at https://github.com/aaronmallen/activeinteractor/blob/main/lib/active_interactor/context/attributes.rb#L80 we can see that under the hood we're simply calling `public_send`

Reply:
Yes I also thought that was not necessary until I saw it the behavior shown in failing specs.

The `public_send` defines attribute with the given key and therefore updates `attributes`

https://github.com/aaronmallen/activeinteractor/blob/c4f6bdd81a8d98b06bb590584dba1cc92ab15d55/lib/active_interactor/context/attributes.rb#L80


While `super` updates the `@table`
https://github.com/aaronmallen/activeinteractor/blob/c4f6bdd81a8d98b06bb590584dba1cc92ab15d55/lib/active_interactor/context/attributes.rb#L82

Reply:
ah that makes sense to me then.  Good catch!